### DISCUSSION TITLE
Problem starting Seq service due to logon failure

### DISCUSSION THREAD

**Discussion by foscojo** [datalust/seq-tickets#2288] [General]
I installed Seq, however, the Seq service fails to start. Error message says "Error: 1069: The service did not start due to a logon failure." In the Seq service properties on the Log On tab, it is set to use This Account ".\SeqDefaultInstance" which doesn't appear to exist.

**Comment by nblumhardt** [comment]
Thanks for your message. The first thing to try will be to re-run the Seq Service Administration applet via the Windows Start Menu. Normally, if the service log on account doesn't exist, this will recreate it.

Let me know if this does the trick, and if not, whether any error messages were shown in the output of the service reconfiguration dialog. Thanks!

**Comment by foscojo** [comment]
Thanks. Unfortunately it didn’t work. Below are the 2 errors I have received. The first was durig install. The second was when I opened the Admin applet as instructed. Attached is the log.

***@***.***

***@***.***

**Reply by nblumhardt** [reply]
Thanks for the additional details. The "The specified service has been marked for deletion" message generally indicates that Windows needs a restart. Restarting Windows, then running through Seq Service Administration again, is the best bet.
